tasks/cephfs/test_volume_client.py

```diff
@@ -11,8 +11,8 @@
 
 class TestVolumeClient(CephFSTestCase):
     # One for looking at the global filesystem, one for being
-    # the VolumeClient, one for mounting the created shares
-    CLIENTS_REQUIRED = 3
+    # the VolumeClient, two for mounting the created shares
+    CLIENTS_REQUIRED = 4
```

Comment:
Need to add a 4th client here: https://github.com/ceph/ceph-qa-suite/blob/master/suites/fs/recovery/clusters/2-remote-clients.yaml

(it's the hook that runs this test in the lab, might not completely make sense out of context but just need to add a client.3 after the client.1,client.2)